frameworks/g-cloud-9/questions/services/backupLocations.yml

```diff
@@ -0,0 +1,21 @@
+name: "Backup locations"
+question: "Where can the service back up to?"
+depends:
+  -
+    "on": lot
+    being:
+      - cloud-hosting
+
+type: checkboxes
+options:
+    -
+        label: "Single, onsite location"
+    -
+        label: "Multiple, onsite locations"
+    -
+        label: "Secure, offsite locations"
```

Comment:
you’re right. This whole questions feels a bit odd:
- ‘secure, offsite location’ implies the other ones aren’t secure, which they could be
- 'Multiple, onsite locations’ is a bit vague. Is the important thing is they are within x-miles but independent of each other?

Checking with @minglis